src/components/avatar/Avatar.scss

```diff
@@ -35,4 +35,14 @@
         display: flex;
         justify-content: center;
     }
+
+    &.large {
```

Comment:
Can this be updated to use SUIT naming conventions? The class name is pretty generic otherwise. Something like `avatar--large`?

Reply:
Not updating the whole file to use SUIT since I see several overrides for `.avatar-initial`, etc., but I'll make this change.